File: froom-backend/src/main/java/com/froom/backend/items/model/dto/ItemDto.java
package com.froom.backend.items.model.dto;

import com.froom.backend.items.model.domain.BodyPart;
import com.froom.backend.items.model.domain.Type;
import lombok.Getter;
import lombok.Setter;
import java.util.UUID;

@Getter
@Setter
public class ItemDto {
    private UUID id;
    private Type type;
    private String color;
    private Double price;
    private BodyPart bodyPart;
    private String size;
}

File: froom-backend/src/main/kotlin/com/froom/froombackend/items/model/dto/ItemDto.kt
package com.froom.froombackend.items.model.dto

import com.froom.froombackend.items.model.domain.BodyPart
import com.froom.froombackend.items.model.domain.Category
import java.util.*

data class ItemDto(
    val uuid: UUID,
    val category: Category,
    val bodyPart: BodyPart,
    val color: List<String>,
    val image: String,
    val imageFormat: String
)